app/controllers/admin/overviews_controller.rb

```diff
@@ -1,6 +1,12 @@
 module Admin
   class OverviewsController < Admin::ApplicationController
     require 'csv'
+    ACCEPTANCE_TEST_USER_ID = 'a5833e7a-a210-4447-904c-df050d198e33'.freeze
```

Comment:
minor picky comment but can these consts be accessed from one location so we don't have to remember to maintain this list too? I'm sure they're used in the cronjob etc.